backend/Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN npm run build
 
 
 # NOTE: Has to be alpine3.16 because newer versions have changed a dependency, causing Prisma to fail
```

Comment:
if this isn't a problem anymore you might want to remove this comment!

Reply:
Ah fair point. I meant to but forgot.